README.md

```diff
@@ -36,6 +36,12 @@ To install,
 yum install -y tmux nmap podman
 ```
 
+One special test case, SR_IOV_OVS_IPv6, requires openvswitch. Without openvswitch installed, this test case will be skipped. To install openvswitch on RHEL 8,
```

Comment:
Should we use the formal name "Open vSwitch" or stick with the command name "openvswitch"?